### DISCUSSION TITLE
Could not use undetected chrome Headless=True

### DISCUSSION THREAD

**Discussion by tamireiny** [ultrafunkamsterdam/undetected-chromedriver#1595] [Q&A]
I am using the following:

- google chrome Version 117.0.5938.132 (Official Build) (64-bit)
- Chrome driver : https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/117.0.5938.132/win64/chromedriver-win64.zip
Undetected chrome driver version 3.5.3
- Selenium 4.13.0
- OS Windows 10 64bits
Trying to do web automation, when using headless False, everything works well, but when using headless True, I get :

"selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:18424 from chrome not reachable"

Any idea how to solve? Thanks!

**Comment by ongbanbanhbao** [comment]
Till now, you need to downgrade chrome 114

**Reply by pudjojotaro** [reply]
is it chrome 114 and lower?
